common/djangoapps/student/models.py

```diff
@@ -3103,3 +3103,15 @@ def should_celebrate_first_section(enrollment):
             return enrollment.celebration.celebrate_first_section
         except CourseEnrollmentCelebration.DoesNotExist:
             return False
+
+
+class UserAccountDisableHistory(TimeStampedModel):
+    """
+    Keeps track of user disable/enable history
+    """
+    user = models.ForeignKey(User, related_name='disable_comment', on_delete=models.CASCADE)
+    comment = models.CharField(max_length=255, help_text=_("Add a reason"))
+    disabled = models.BooleanField(default=True)
```

Comment:
Add another field to record the user who disabled the account.

Reply:
Done